src/planner/linux/steam_deck.rs

```diff
@@ -188,7 +189,7 @@ impl Planner for SteamDeck {
             Type=oneshot\n\
             RemainAfterExit=yes\n\
             ExecStart=/usr/bin/systemctl daemon-reload\n\
-            ExecStart=/usr/bin/systemctl restart --no-block sockets.target timers.target multi-user.target\n\
+            ExecStart=/usr/bin/systemctl start --no-block sockets.target timers.target multi-user.target\n\
```

Comment:
Is there a reason we don't start the desired services directly instead of this spray'n'pray approach? This is certainly better than restarting, but could still have potentially adverse effects.

Reply:
For some reason I think I tried this and it didn't work, however I don't see a commit so I am going to test since I agree, that would be way better.